src/main/java/SpringJPA/Model/User.java

```diff
@@ -51,6 +55,25 @@ public User(String user, String pass, String mail, UserType role, long apiCallCo
         this.role = role;
         this.apiCallCount = apiCallCount;
         this.apiCallLimit = apiCallLimit;
+        this.startTime = System.currentTimeMillis();
```

Comment:
Change it so new users don't automatically start a trial please

Reply:
Done.